.circleci/config.yml

```diff
@@ -87,7 +87,8 @@ commands:
           name: Install Arlecchino
           command: |
             cd
-            export GIT_SSH_COMMAND="ssh -o IdentitiesOnly=yes -i ~/.ssh/id_rsa_2a113f169fab90346b5128f5b73ad16d"
+            echo 'export GIT_SSH_COMMAND="ssh -o IdentitiesOnly=yes -i /home/circleci/.ssh/id_rsa_2a113f169fab90346b5128f5b73ad16d"' > /tmp/arlecchino.sh
```

Comment:
Consider having a getarlecchino script in readies (so it could also be run manually if needed)

Reply:
@oshadmi We have it alright (getrlec) in readies/master, but it won't make this dance go away until we OSS arlecchino.